recipes/archicad-apidevkit/all/conanfile.py

```diff
@@ -0,0 +1,36 @@
+from conan import ConanFile
+from conan.tools.files import get
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.00"
```

Comment:
```suggestion
required_conan_version = ">=1.46.0"
```

copy requires 1.46.0 https://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md